pkg/kubeletclient/kubeletclient.go

```diff
@@ -25,6 +25,9 @@ const (
 func GetResourceClient(kubeletSocket string) (types.ResourceClient, error) {
 	// If Kubelet resource API endpoint exist use that by default
 	// Or else fallback with checkpoint file
+	if kubeletSocket == "" {
```

Comment:
this if should probably be placed before the above comment, which seems to reference the next if below

Reply:
well, this is not big issue I suppose, because this code (adding default kubeletSocket) is also for second 'if' statement.
To me, at least, it makes sense.

Reply:
moved the comment